tests/unit/constraints/test_tabular.py

```diff
@@ -1274,3 +1274,174 @@ def test_reverse_transform(self):
             'c': [5, 7, 9]
         })
         pd.testing.assert_frame_equal(expected_out, out)
+
+
+class TestRounding():
+
+    def test___init__(self):
```

Comment:
We should add tests cases that cover the validations: digits > 15 and tolerance < digits